shell/browser/electron_browser_client.cc

```diff
@@ -1540,11 +1540,15 @@ bool ElectronBrowserClient::PreSpawnChild(sandbox::TargetPolicy* policy,
                                           sandbox::mojom::Sandbox sandbox_type,
                                           ChildSpawnFlags flags) {
   // Allow crashpad to communicate via named pipe.
+  if (policy->GetConfig()->IsConfigured())
+    return true;
```

Comment:
It looks like upstream perhaps DCHECKs !IsConfigured here: https://source.chromium.org/chromium/chromium/src/+/main:sandbox/policy/win/sandbox_win.cc;l=330;drc=6cde7d618878230861163e0d68ba652549ba6a45

what's the reasoning behind early-outing in this case? What is the effect?

Reply:
When I was looking at the CL, I think the if we try to add a rule to the config after the config has been configured/frozen, it will throw a DCHECK here: https://chromium-review.googlesource.com/c/chromium/src/+/3558444

This may not be necessary here though, since we're just calling it once for crashpad. I'll remove this!

Reply:
it should throw a DCHECK in that case, since it would be failing to add this policy, causing crashpad not to work on Windows